webcompat/templates/shared/footer.html

```diff
@@ -45,7 +45,7 @@
             <a class="footer-item-link" href="/contributors/report-bug">Report a Bug</a>
           </li>
           <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_bug_reproduce' %}active{% endif %}">
-            <a class="footer-item-link" href="/contributors/reproduce-bug">Reproduce a Bug</a>
+            <a class="footer-item-link" href="/contributors/triage-bug">Triage a bug</a>
```

Comment:
```suggestion
            <a class="footer-item-link" href="/contributors/reproduce-bug">Triage a bug</a>
```

if the link name is changed, it will probably break.